wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt

```diff
@@ -328,6 +328,61 @@ class KotlinGeneratorTest {
     assertEquals(expected, repoBuilder.generateGrpcKotlin("routeguide.RouteGuide"))
   }
 
+  @Test fun multipleRpcsAsSingleMethodInterface() {
+    val repoBuilder = RepoBuilder()
+        .add("routeguide.proto", """
+          |package routeguide;
+          |
+          |service RouteGuide {
+          |  rpc GetFeature(Point) returns (Feature) {}
+          |  rpc RouteChat(stream RouteNote) returns (stream RouteNote) {}
+          |}
+          |$pointMessage
+          |$featureMessage
+          |$routeNoteMessage
+          |""".trimMargin())
+
+    val expectedGetFeature = """
+          |package routeguide
+          |
+          |import com.squareup.wire.Service
+          |import com.squareup.wire.WireRpc
+          |
+          |interface RouteGuideGetFeature : Service {
+          |    @WireRpc(
+          |            path = "/routeguide.RouteGuide/GetFeature",
+          |            requestAdapter = "routeguide.Point#ADAPTER",
+          |            responseAdapter = "routeguide.Feature#ADAPTER"
+          |    )
+          |    suspend fun GetFeature(request: Point): Feature
```

Comment:
this looks good

I wanna have a blocking option somehow

Reply:
it can come later I guess